Gemfile

```diff
@@ -7,12 +7,11 @@ gem 'rake', '~> 10.3'
 
 # DLSS gems
 gem 'dor-services', '~> 5.3'
-gem 'lyber-core', '~> 3.2', '>=3.2.4'
+gem 'lyber-core', '~> 4.0'
 gem 'robot-controller', '~> 1.0' # requires Resque
 
 group :development, :test do
   gem 'rspec'
   gem 'yard'
   gem 'dlss-capistrano'
-  gem 'holepicker', '~> 0.3', '>= 0.3.3'
 end
```

Comment:
Could this group be split up?  A 'deployment' group could split out the capistrano dependencies so they don't creep into development deployments.  rspec isn't required for a development deployment.